### DISCUSSION TITLE
How to set up for subcutaneous administrations?

### DISCUSSION THREAD

**Discussion by Achineseman** [Open-Systems-Pharmacology/Forum#958] [Questions & Problems] [answered]
<img width="367" alt="屏幕截图 2021-08-22 233545" src="https://user-images.githubusercontent.com/81560924/130361125-b9f5e1db-cc31-4979-98f9-ae202df5b91a.png">
<img width="515" alt="屏幕截图 2021-08-22 233611" src="https://user-images.githubusercontent.com/81560924/130361131-c703d17b-1a0f-4d89-9f2a-91695cbe910a.png">

Did I do the right thing?

**Comment by StephanSchaller** [comment, marked as answer]
For a simplified SC dosing of insulin, yes. But as Insulin is a peptide that may form hexamers in the sc depot you might need to model the SC absorption of insulin mechanistically, especially if you are dosing insulins other than the very fast acting analogs.

Please see:
https://link.springer.com/article/10.1007/BF01059088
https://journals.sagepub.com/doi/pdf/10.1177/193229680700100317

...and for a simplified implementation in PK-Sim:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3828004/
